bin/export.py

```diff
@@ -104,37 +104,32 @@ def build_samples_zip(problems, statement_language):
 def build_problem_zip(problem, output, statement_language):
     """Make DOMjudge ZIP file for specified problem."""
 
+    deprecated = [  # may be removed at some point.
+        'domjudge-problem.ini',
+    ]
+
     files = [
-        ('domjudge-problem.ini', False),  # DEPRECATED, may be removed at some point.
         ('problem.yaml', True),
         ('problem_statement/*', True),
-        ('data/secret/**/*.in', True),
-        ('data/secret/**/*.ans', True),
         ('submissions/accepted/**/*', True),
         ('submissions/*/**/*', False),
-        ('attachments/**/*', False),
+        ('attachments/**/*', problem.interactive),
     ]
 
-    for ext in config.KNOWN_DATA_EXTENSIONS:
-        files += [
-            (f'data/sample/**/*{ext}', False),
-            (f'data/secret/**/*{ext}', False),
-        ]
+    testcases = [
+        ('data/secret/**/*.in', True),
+        ('data/sample/**/*.in', not problem.interactive),
```

Comment:
Hmm I think by default we should include the sample inputs, so they can actually be run as samples?
Otherwise we should generatlly disallow `sample/*.{in,ans}` for interactive problems and make a big warning here if they are present.

Reply:
They are now included (unlike before) but not required for interactive problems?

Reply:
Oh right I was completely misreading the code. Lack of sleep is showing up. All's good to go them :)